tests/test_integrity.py

```diff
@@ -334,3 +334,24 @@ def test_repository_license(self):
                     OBO_TO_SPDX[obo_license],
                     msg="OBO Foundry license annotation does not match GitHub license",
                 )
+
+    def test_contribution_guidelines(self):
+        """Test that a contribution guidelines document is available in an expected location/format."""
+        for prefix, data in self.ontologies.items():
+            repository = data["repository"]
+            if not repository.startswith("https://github.com"):
+                continue
+            r = repository.removeprefix("https://github.com/").rstrip("/")
+            github_data = self._get_github_data(prefix)
+            default_branch = github_data["default_branch"]
+            paths = [
+                f"https://github.com/{r}/blob/{default_branch}/CONTRIBUTING.md",
+                f"https://github.com/{r}/blob/{default_branch}/docs/CONTRIBUTING.md",
+                f"https://github.com/{r}/blob/{default_branch}/.github/CONTRIBUTING.md",
```

Comment:
This will reject any repository with just a `CONTRIBUTING` or `CONTRIBUTING .rst` correct? is that we want, to enforce `CONTRIBUTING.md` only?

Reply:
If there's no formatting then i am insulted that they care so little for potential contributors